app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,26 @@
+class MessagesController < ApplicationController
+  def new
+    @message = Message.new
+  end
+
+  def create
+    @message = Message.new(message_params)
+    if @message.save
+      redirect_to messages_path
+    else
+      render 'new'
+    end
+  end
+
+  def index
+    @message = Message.order('RANDOM()').first
+    render json: { message: @message.text }
+    puts
```

Comment:
- [ ] Great effort working on your project, but while trying to the directory where you have the messages API endpoint it throws an error and this error makes it difficult to navigate there and all the messages are not shown. Also, this makes the random messages not render on the browser on the front end side. I would kindly suggest you work on it and ensure everything works fine.

Video Recording 👇 


https://github.com/JuanDBta/hello-rails-back-end/assets/78634841/89cb60b2-d5b1-4c68-9a1a-46b9ddc861b9

Reply:
Dear reviewer @brainconnect93:

Thank you for your review. I've checked the app with my partners on their local machines and is working fine.  Maybe you missed something in the database.yml file or running a migration like rails db:create or bin/rails db:seed.

Reply:
@JuanDBta, am so sorry about the mistake, I checked through the whole project again now and I realized the mistake from my machine PGAdmin and now I see the project is working fine.
Am so sorry for the inconvenience my mistake might have caused you. 
@felix, Thanks for approving the project.